### PR TITLE
[13.0][ADD] product_attribute_archive

### PR DESCRIPTION
**Pull request by AlvaroTForgeFlow**
Backport from v14 to v13 of the product_attribute_archive module.

### CODE REVIEW

**Comment by AlvaroTForgeFlow** [comment]
Hi @JordiBForgeFlow  , could I have a review here?

**Comment by HviorForgeFlow** [comment]
@OCA/product-maintainers can you  approve running workflow, thanks!

**Review by rousseldenis — APPROVED** [review]
Code review

File: product_attribute_archive/views/product_attribute.xml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            <xpath expr="//group[@name='main_fields']" position="before">
+                    <field name="active" invisible="1" />
+                    <widget
+                    name="web_ribbon"
+                    title="Archived"
+                    bg_color="bg-danger"
+                    attrs="{'invisible': [('active', '=', True)]}"
+                />
+            </xpath>
+            <field name="active" optional="hide" />
```
Comment (HviorForgeFlow) [inline product_attribute_archive/views/product_attribute.xml]: Unable to show column active on product.attribute form, you should apply to place that column inside the Attribute Value tree

```suggestion
            <xpath expr="//field[@name='value_ids']/tree" position="inside">
              <field name="active" optional="hide"/>
            </xpath>
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
@AlvaroTForgeFlow

File: product_attribute_archive/views/product_attribute.xml (hunk 1 above)
Comment (HviorForgeFlow) [reply product_attribute_archive/views/product_attribute.xml]: Rebase required to fix pre-commit

**Review by HviorForgeFlow — APPROVED** [review]
Functional review LGTM :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
code LGTM

**Pull request merged**